mewarpx/mewarpx/setups_store/diode_setup.py

```diff
@@ -167,24 +167,27 @@ class DiodeRun_V1(object):
     # steps between doing load-balancing. If <1 load balancing will not be done
     LOAD_BALANCE_INTERVALS = 0
 
-    def __init__(self, dim=1, rz=False, **kwargs):
+    def __init__(self, **kwargs):
         for kw in list(kwargs.keys()):
             setattr(self, kw, kwargs[kw])
```

Comment:
Small suggestion that I should've made when this code was first written, but I didn't know this at the time. It's faster to iterate through all the items of the dictionary than to just iterate through the keys.
```suggestion
        for kw, value in kwargs.items():
            setattr(self, kw, value)
```